src/bundle/DependencyInjection/EzPlatformRichTextExtension.php

```diff
@@ -177,4 +182,80 @@ private function validateCustomTemplatesConfig(
             }
         }
     }
+
+    /**
+     * Validate presence of inline Custom Tags in Toolbars.
+     *
+     * @param array $availableSiteAccesses a list of available SiteAccesses
+     * @param array $customTagsConfig Custom Tags configuration
+     * @param \Symfony\Component\DependencyInjection\ContainerBuilder $container
+     */
+    private function validateInlineCustomTagToolbarsConfig(
+        array $availableSiteAccesses,
+        array $customTagsConfig,
+        ContainerBuilder $container
+    ): void {
+        $customTags = $this->getInlineCustomTags($customTagsConfig);
+        foreach ($this->getToolbarsBySiteAccess($availableSiteAccesses, $container) as $siteAccess => $toolbar) {
+            foreach ($toolbar as $toolbarName => $toolbarContent) {
+                $this->checkForInlineTagsInToolbar($toolbarName, $toolbarContent, $customTags, $siteAccess);
+            }
+        }
+    }
+
+    /**
+     * @return iterable<array> Iterable containing arrays with toolbars and their buttons
+     */
+    private function getToolbarsBySiteAccess(array $availableSiteAccesses, ContainerBuilder $container): iterable
+    {
+        foreach ($availableSiteAccesses as $siteAccessName) {
+            $paramName = "ezsettings.{$siteAccessName}.fieldtypes.ezrichtext.toolbars";
+            if (!$container->hasParameter($paramName)) {
+                continue;
+            }
+
+            yield $paramName => $container->getParameter($paramName);
+        }
+    }
+
+    /**
+     * @return string[]
+     */
+    private function getInlineCustomTags(array $customTagsConfig): array
+    {
+        $customTags = array_filter(
+            $customTagsConfig,
+            static function (array $customTag): bool {
+                return $customTag['is_inline'] ?? false;
+            }
+        );
+
+        return array_keys($customTags);
+    }
+
+    private function checkForInlineTagsInToolbar(
+        string $toolbarName,
+        array $toolbarContent,
+        array $customTags,
+        string $siteAccess
+    ): void {
+        // "text" toolbar is the only one that can contain inline tags
+        if ($toolbarName === 'text') {
```

Comment:
Could you please define const. representing toolbar identifiers that can contain inline custom tags?

Reply:
Done.